pkg/virt-handler/vm_test.go

```diff
@@ -1151,6 +1151,51 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(blockMigrate).To(BeTrue())
 			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared HostDisk")))
 		})
+
+		It("should block migrations if the pod network is connected via a binding method different than masquerade?", func() {
```

Comment:
I don't think the question mark should be there, but I'm fine leaving it there for fun.

Reply:
Corrected it now :smile: